addons/account/models/account_reconcile_model.py

```diff
@@ -185,6 +165,25 @@ def _onchange_match_total_amount_param(self):
     # RECONCILIATION PROCESS
     ####################################################
 
+    def _make_line_for_widget(self, line):
+        if not line.get('reconcileModelId'):
+            line['label'] = line.pop('name')
+            line['id'] = 'createLine' + str(random.randint(0, 999999))
```

Comment:
this is dubious. I'm expecting the Web client to assign the virtual id of the line it will create

Reply:
This is it, the web client doesn't create the line anymore. The client used `_.uniqueId('createLine')` from the [Underscore.js](https://underscorejs.org) library. I can't really do that anymore because I have no idea what the client is up to.